agents_infra/install

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+# Setup build environment
+apt update && apt install -y ruby ruby-dev build-essential ansible rsync sudo
+gem install --no-document dotenv:2.8.1 fpm:1.15.1
+
+if [ -f .env.sh ]; then
+  source .env.sh
+fi
+
+bash scripts/run-ansible.sh
```

Comment:
Consider making the script path relative to the repo root using `$(dirname "$0")` or checking for script existence first.